content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -107,14 +110,14 @@ metadata:
   labels:
     tags.datadoghq.com/env: "<ENV>"
     tags.datadoghq.com/service: "<SERVICE>"
-    # tags.datadoghq.com/version: "<VERSION>" #Uncomment this line to manually tag your version
+    tags.datadoghq.com/version: "<VERSION>" #Comment out this line to use automatic version tagging in APM
```

Comment:
Commenting out this line will result in no version being set for logs/metrics though.

Reply:
I removed messaging to comment this out